### ISSUE TITLE
Allow running on in progress AOCs

### ISSUE THREAD

**Issue by ThinkSalat** [beakerandjake/advent-of-code-runner#240]
I've already submitted answers to some of the problems and it appears to only work on a completely fresh start. Is it possible to allow for in progress calendars? I can't submit day 1 for example, because it's already been submitted. Trying to run day 1 part 2 throws an error saying I need to submit part 1 first.

**Comment by beakerandjake** [comment]
Hey, thanks for submitting an issue! I see what you mean, it's something I've been meaning to fix (see #20). I hadn't encountered it because I've always used advent-of-code-runner each year from day 1. I will add in a new command that allows you to set the answer to a problem you've solved outside of advent-of-code-runner, allowing you to use the package for in progress calendars. I'll start on a fix first thing tomorrow! 

What's happening is that advent-of-code-runner stores your progress in a json file at the root of your repository called `aocr-data.json`. It doesn't know about the puzzle you've solved because the answer isn't in that file. 

### Manual Hotfix (for version 1.6.1) 

As a temporary work around until I get that new command in you can manually edit this file. Normally you shouldn't edit it because if something gets set incorrectly it could cause unexpected errors. 

Here is the default `aocr-data.json` created when you run the `init` command: 

```
{
  "version": "1.6.1",
  "year": 2023,
  "description": "This file is where your progress is stored. It should be committed to source control. You shouldn't edit this file manually, it could cause unexpected errors.",
  "puzzles": []
}
```

To add puzzles you've already completed just add new object in the `puzzles` array. You need to add in a new object for each puzzle you've solved: 

```
{
  "id": "20220101",
  "fastestRuntimeNs": 99999999999,
  "incorrectAnswers": [],
  "correctAnswer": "73211"
}
```
The properties you will need to set: 

- `id`: This is a **string** formatted: year day level. So year 2023 day 5 level 2 becomes: `"20230502". Year 2023 day 1 level 1 becomes: "20230101". 
- `correctAnswer`: This is a **string**. You would put the correct answer in this field. 

You can ignore `incorrectAnswers` and `fastestRuntimeNs` for now (when you run a `solve` command for a completed puzzle it will override `fastestRuntimeNs` with an updated value)

So for now you can add in entries for any puzzles you've completed to get your repository in sync. Here's an example `aocr-data.json` where I manually added in entries for day 1 level 1 and day 2 level 2. 

```
{
  "version": "1.6.1",
  "year": 2023,
  "description": "This file is where your progress is stored. It should be committed to source control. You shouldn't edit this file manually, it could cause unexpected errors.",
  "puzzles": [
    {
      "id": "20230101",
      "fastestRuntimeNs": 99999999999,
      "incorrectAnswers": [],
      "correctAnswer": "DAY ONE LEVEL ONE ANSWER"
    },
    {
      "id": "20230102",
      "fastestRuntimeNs": 99999999999,
      "incorrectAnswers": [],
      "correctAnswer": "DAY ONE LEVEL TWO ANSWER"
    }
  ]
}
```

**Comment by ThinkSalat** [comment]
Amazing, thank you so much! I'm only up to day 5 part 2 so it won't be too much to manually add my previous day's work. Thanks for such an awesome project!

**Comment by beakerandjake** [comment]
Awesome glad you like it! Let me know if you have any issues with the manual hot fix. Keep an eye out for an update with the new command, I should be able to get it published tomorrow.

**Comment by beakerandjake** [comment]
Just released 1.7.0 which adds the `import` command!
